content/wiki/Coaching-a-team.md

```diff
@@ -0,0 +1,91 @@
+---
+title: "Coaching a team"
+date: 2022-12-02T11:50:27+02:00
+---
+In programming contests a team has a coach. A coach registers a team for the contest and is the main contact between 
+the organisers of the contest and the team. There are other things a coach can do for a team. This article will
```

Comment:
I would give this sentence a headline of its own. So that if you look at the layout of the page you see a complete list of headings of what a coach can do, not only the "extra" things a coach can do if you know what I mean. This sentence could also (later) be expanded upon e.g. about what registering a team means.

Reply:
This is actually already done by the template engine. The title is always added as the first header. It is a bit confusing if you don't know this quirk of the template. 
![image](https://user-images.githubusercontent.com/5221158/205499897-6c7c5598-82e4-4f48-9371-9c843a00ec9f.png)